wgsl/index.bs

```diff
@@ -120,9 +120,9 @@ This specification does not prescribe a particular encoding for that text.
 
 ## Comments ## {#comments}
 
-Comments begin with `//` and continue to the end of the current line. There are no multi-line comments.
-
-TODO: What indicates the end of a line?  (E.g. A line ends at the next linefeed or at the end of the program)
+Line-comments begin with `//` and continue to the end of the current line or file.
+Block-comments begin with `/*` and continue to the end of the first subsequent `*/`. (no nesting)
```

Comment:
Should we talk about allowing the nesting?
If we say it's not allowed today, we will not be able to revise this decision later - https://softwareengineering.stackexchange.com/questions/81072/why-do-most-programming-languages-not-nest-block-comments#comment492185_81074